README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
```

Comment:
I envision devex changes for this @adambkaplan @siamaksade @bparees, updating our existing image change controller and the existing annotation it has for generic k8s object, to include support for both buildv2 and tekton

Certainly we can discuss the pros/cons with @sbose78 of them creating their own controller for buildv2 and this, but IMO the former idea would cost substantially less

Reply:
https://github.com/redhat-developer/build/issues/52